seed/challenges/01-responsive-web-design/basic-css.json

```diff
@@ -1160,6 +1534,7 @@
         "<code>id</code> attributes should be unique. Browsers won't enforce this, but it is a widely agreed upon best practice. So please don't give more than one element the same <code>id</code> attribute.",
```

Comment:
I think since classes were introduced earlier, we should compare an id to a class in the instructions here. Something to the effect of:
>Like classes, you can use an <code>id</code> to style an element but an <code>id</code> is not reusable and should only be applied to one element. An <code>id</code> also has a higher specificity (importance) than a class so if both are applied to the same element and have conflicting styles, the styles of the <code>id</code> will be applied.